Makefile.am

```diff
@@ -242,10 +242,16 @@ compliant:
 	git diff --quiet --exit-code src; ret=$$?; \
 	if [ $$ret -eq 1 ]; then \
 		echo "Code style issues found. Run 'git diff' to view issues."; \
+	fi; \
+	OUT=$$(git grep if\ \(.*\)$$ -- '*.c'); ret2=$$?; \
+	if [ $$ret2 -eq 0 ]; then \
+		echo "Missing brackets in single line if:"; \
+		echo "$$OUT"; \
 	elif [ $$ret -eq 0 ]; then \
 		echo "No code style issues found."; \
+		exit 0; \
```

Comment:
What if clang-format makes changes in the source files and there are no missing brackets in single if? Based on this logic both this messages would be shown at the same time, wouldn't they?
"Code style issues found. Run 'git diff' to view issues."
"No code style issues found."

Reply:
Yes, probably an `exit 1` should be added above. I'll add a separate comment.

Reply:
there's no need to add an exit 1.
I'm checking if ret -eq 0 in an elif. so if we reach exit 0, ret == 0 and ret2 == 0, so no errors were found.

Reply:
Oh, sorry, I didn't notice that a new "ret2" variable was introduced... You're correct. The logic looks okay.

Reply:
You are right, now that I see it again I don't even know why I thought it wouldn't work, I must have misread some variable or something. I'll pay more attention in the future. 
So anyway, looks good ;)